vector/src/main/java/im/vector/app/core/di/FragmentModule.kt

```diff
@@ -515,6 +516,11 @@ interface FragmentModule {
     @FragmentKey(FtueAuthPhoneEntryFragment::class)
     fun bindFtueAuthPhoneEntryFragment(fragment: FtueAuthPhoneEntryFragment): Fragment
 
+    @Binds
```

Comment:
To avoid declaring explicitely bindings for fragment, we can now annotate the `Fragment` by `@AndroidEntryPoint`.

Reply:
No problem we can keep this way if this is a planned work. I thought it would make less work on the refactoring if we apply this strategy for each new `Fragment`.